src/main/util/consts.py

```diff
@@ -66,12 +75,15 @@ class DEFAULT_VALUE(Enum):
     EVENT_TYPE = nan
     EVENT_DATA = nan
 
+    # todo: add tests
```

Comment:
Is it works well? Can we add tests in the other PR?

Reply:
Yeah, let's leave it now :)